core/workflow.go

```diff
@@ -19,13 +19,16 @@ limitations under the License.
 
 package core
 
+// WorkflowState int type
```

Comment:
Here and below at 31 can be changed to be similar to what was stated above re: Task states.